Assets/MRTK/Examples/Demos/StandardShader/Scenes/ClippingInstancedExamples.unity

```diff
@@ -0,0 +1,8172 @@
+%YAML 1.1
```

Comment:
Love this scene! I saw some minor performance issues when running on HoloLens 2. But since this scene is geared towards being a stress test, I think it's fine. I took a peek in the frame debugger and verified everything is instancing as expected.

Reply:
Hmmm, I was able to get ~400 draw calls down to ~20 on HL2 and saw a steady 60 on the Visual Profiler. I'll double check performance on device in the examples.

Reply:
New changes show the frame rate at a steady 60 for me. Please confirm and resolve if you see the same. =)